src/store/thunks/posts.js

```diff
@@ -1,18 +1,41 @@
 import { postsFetched } from "store/actions/posts";
 import { db } from "database";
 
-export const fetchPosts = () => (dispatch) => {
-  db.collection("posts")
-    .orderBy("created_at", "desc")
-    .get()
-    .then((snapshot) => {
-      const postArray = [];
-      snapshot.forEach((post) => {
-        const postObj = post.data();
-        postObj.id = post.id;
-        postArray.push(postObj);
-      });
-      dispatch(postsFetched(postArray));
-    })
-    .catch(console.error);
+export const fetchPosts = (sortBy, timeframe) => (dispatch) => {
+  if (sortBy === "newest") {
+    db.collection("posts")
```

Comment:
Good idea to stop using .then() and start using async await from here on out

Reply:
???? where is this using .then?